app/src/resources/camera/usePreviewImage.ts

```diff
@@ -1,44 +1,69 @@
-import { useMemo, useState } from 'react'
+import { useEffect, useMemo, useState } from 'react'
 
-import { useCapturePreviewImage } from '@opentrons/react-api-client'
-
-import { MIME_TYPES } from '../dataFiles/constants'
+import {
+  useCapturePreviewImage,
+  useCapturePreviewImageToRun,
+} from '@opentrons/react-api-client'
 
 import type { CameraImageSettings } from '@opentrons/api-client'
 
 export interface UsePreviewImageResult {
-  imgPath: string | undefined
+  imgPath: string | null
   isLoading: boolean
   takePhoto: () => void
 }
 
 export function usePreviewImage(
-  settings: CameraImageSettings
+  settings: CameraImageSettings,
+  runId: string | null
 ): UsePreviewImageResult {
-  const [isLoading, setIsLoading] = useState(false)
-  const { data, refetch } = useCapturePreviewImage(settings, {
-    onSettled: () => {
-      setIsLoading(false)
+  const [imageBlob, setImageBlob] = useState<Blob | null>(null)
+  const callbacks = {
```

Comment:
```suggestion
  const capturePreviewImageCbs = {
```
Nit, but it helps keep clear what "callbacks" means exactly in the lexical scope of this hook without needed to dig around internals.